src/librustdoc/html/render.rs

```diff
@@ -2112,11 +2162,12 @@ fn item_struct(w: &mut fmt::Formatter, cx: &Context, it: &clean::Item,
         if fields.peek().is_some() {
             try!(write!(w, "<h2 class='fields'>Fields</h2>\n<table>"));
             for field in fields {
-                try!(write!(w, "<tr class='stab {stab}'>
-                                  <td id='structfield.{name}'>\
-                                    <code>{name}</code></td><td>",
-                            stab = field.stability_class(),
-                            name = field.name.as_ref().unwrap()));
+                let name = field.name.as_ref().unwrap();
+                try!(with_unique_id(format!("structfield.{}", name), |id|
```

Comment:
I think that ids like this don't need to be altered because there shouldn't ever be more than one struct on a page?